playbooks/cloud-provider-openstack-acceptance-test-keystone-authentication-authorization/run.yaml

```diff
@@ -167,13 +166,13 @@
 
           # NOTE: k8s+keystone webhook test case need the OS_DOMAIN_NAME env
           export OS_DOMAIN_NAME=Default
-          "$kubectl" config set-credentials openstackuser --auth-provider=openstack
-          "$kubectl" config set-context --cluster=local --user=openstackuser openstackuser@local
-          "$kubectl" config use-context openstackuser@local
-          if ! "$kubectl" get pods; then
-              echo "Testing kubernetes+keystone authentication and authorization failed!"
+          '{{ kubectl }}' config set-credentials openstackuser --auth-provider=openstack
+          '{{ kubectl }}' config set-context --cluster=local --user=openstackuser openstackuser@local
+          '{{ kubectl }}' config use-context openstackuser@local
+          if ! '{{ kubectl }}' get pods; then
+              echo "Testing kubernetes+keystone authentication and authorizatio failed!"
```

Comment:
sorry, I made a typo authorizatio and I have correct in my last PR, but you change it back here :-P